src/forms/TimeInput.jsx

```diff
@@ -2,6 +2,40 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
 
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import SelectInput from './SelectInput';
+
+export const HOURS_INPUT_CLASS = 'timeInput-hours';
+export const MINUTES_INPUT_CLASS = 'timeInput-minutes';
+export const MERIDIAN_INPUT_CLASS = 'timeInput-meridian';
+
+export const HOURS = 'hours';
+export const MINUTES = 'minutes';
+export const MERIDIAN = 'meridian';
```

Comment:
I think I would avoid abstracting the keys into consts - they're intuitively named and don't need to be used outside of this component code, but fine to leave in if you found it useful to use them. Sort of makes object shorthand and destructuring more awkward.